app/uk/gov/hmrc/gform/views/form/addToList.scala.html

```diff
@@ -58,9 +59,13 @@
 
 @heading = @{repeater.title.value}
 
-@buttonLabel = @{
-    messages("button.saveAndContinue")
-}
+@button  = @{formTemplate.authConfig match {
+                    case Anonymous =>
+                        messages("button.continue")
+                    case _ =>
+                        messages("button.saveAndContinue")
+                    }
+            }
```

Comment:
There is `continueLabelKey` method for this on `MaterialisedRetrievals`

Reply:
Changed.